pkg/brokerapi/service_binding.go

```diff
@@ -44,9 +44,4 @@ type CreateServiceBindingResponse struct {
 
 // Credential represents connection details, username, and password that are
 // provisioned when a consumer binds to a service instance
-type Credential struct {
-	Hostname string `json:"hostname"`
-	Port     string `json:"port"`
-	Username string `json:"username"`
-	Password string `json:"password"`
-}
+type Credential map[string]interface{}
```

Comment:
My original thinking was to do a delayed parsing so that if people wanted to deal with it as a golang object (instead of a string, where they then have to Unmarshal it before they can use it) keeping it as an interface{} saves that hit.